src/cascadia/TerminalSettingsEditor/Profiles_Advanced.xaml

```diff
@@ -117,6 +117,69 @@
                 </StackPanel>
             </local:SettingContainer>
 
+            <!--  Bell Sound  -->
+            <local:SettingContainer x:Uid="Profile_BellSound"
+                                    ClearSettingValue="{x:Bind Profile.ClearBellSound}"
+                                    CurrentValue="{x:Bind Profile.BellSoundPreview, Mode=OneWay}"
+                                    HasSettingValue="{x:Bind Profile.HasBellSound, Mode=OneWay}"
+                                    SettingOverrideSource="{x:Bind Profile.BellSoundOverrideSource, Mode=OneWay}"
+                                    Style="{StaticResource ExpanderSettingContainerStyle}">
+                <StackPanel Spacing="10">
+                    <TextBlock x:Uid="Profile_BellSoundInfo"
+                               Style="{StaticResource DisclaimerStyle}" />
+                    <ItemsControl ItemsSource="{x:Bind Profile.CurrentBellSounds, Mode=OneWay}">
+                        <ItemsControl.ItemsPanel>
+                            <ItemsPanelTemplate>
+                                <StackPanel Spacing="5" />
+                            </ItemsPanelTemplate>
+                        </ItemsControl.ItemsPanel>
+                        <ItemsControl.ItemTemplate>
+                            <DataTemplate x:DataType="local:BellSoundViewModel">
+                                <Grid TabFocusNavigation="Local">
+                                    <Grid.ColumnDefinitions>
+                                        <ColumnDefinition Width="300" />
+                                        <ColumnDefinition Width="Auto" />
+                                        <ColumnDefinition Width="Auto" />
+                                    </Grid.ColumnDefinitions>
+                                    <TextBlock x:Uid="Profile_BellSoundPath"
+                                               Grid.Column="0"
+                                               Style="{StaticResource TextBlockSettingStyle}"
+                                               Text="{x:Bind Path=Path, Mode=TwoWay}" />
+                                    <Button x:Uid="Profile_BellSoundAudioPreview"
+                                            Grid.Column="1"
+                                            Margin="5,0,0,0"
+                                            VerticalAlignment="Stretch"
+                                            Click="BellSoundAudioPreview_Click"
+                                            Style="{StaticResource BrowseButtonStyle}"
+                                            Tag="{Binding Mode=OneWay}">
+                                        <FontIcon FontSize="{StaticResource StandardIconSize}"
+                                                  Glyph="&#xE767;" />
+                                    </Button>
+                                    <Button x:Uid="Profile_BellSoundDelete"

```

Comment:
This probably needs an automation property and tooltip since we're just using the icon